Automattic/newspack-blocks
# Columns Block: Remove update block output

### All Submissions:


### Changes proposed in this Pull Request:

This PR removes the modified Columns block output, since `calc()` is stripped out on save if the current user's account does not have `unfiltered_html` (by default, only Administrators have this). 

It also adjusts the border block styles to continue to display correctly. 

This PR should be tested with the complementary theme PR, https://github.com/Automattic/newspack-theme/pull/685. 

Note: This update will break columns on existing sites, since it changes the output -- they'll display correctly, but the block will throw an error in the editor. Ideally before committing this update we'll have a script to go with it to correct the columns blocks, and roll it out to sites one at a time to make sure we're not introducing issues. 

Closes #321.

### How to test the changes in this Pull Request:

1. Apply this PR and the related PR https://github.com/Automattic/newspack-theme/pull/685 to the theme. Run `npm run build` for both.
2. Add columns to the editor with various settings (in column count, width, and using the available styles). In the columns, add paragraph blocks with backgrounds; it's the easiest way to see where the edges of the columns are.
3. On the front-end, test different screen sizes, and make sure the columns are behaving as expected (not wrapping until the window is less than 600px wide).
4. Specifically for the theme PR, navigate to Customizer > Style Packs and switch to Style 3; confirm that the columns block with the border uses a dark dotted line on the front-end and in the editor, rather than a light grey border.
5. Specifically for the theme PR, navigate to Customizer > Style Packs and switch to Style 5; confirm that the columns block with the border uses a dark border on the front-end and in the editor, rather than a light grey border.

### Other information:
